lib/gtfs_server/agencies/trip.ex

```diff
@@ -0,0 +1,44 @@
+defmodule GtfsServer.Agencies.Trip do
```

Comment:
I got lost on this one. I'm not sure how to write a context for a schema/table like this that is a nested association from the top level resource.